WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -781,25 +781,14 @@ private extension ProductFormViewController {
 
     /// Updates table viewmodel and datasource and attempts to animate cell deletion/insertion.
     ///
-    func reloadLinkedPromoCellAnimated() {
-        let indexPathBeforeReload = findLinkedPromoCellIndexPath()
+    func reloadLinkedPromoCell() {
         tableViewModel = DefaultProductFormTableViewModel(product: viewModel.productModel,
                                                           actionsFactory: viewModel.actionsFactory,
                                                           currency: currency,
                                                           isDescriptionAIEnabled: aiEligibilityChecker.isFeatureEnabled(.description))
-        let indexPathAfterReload = findLinkedPromoCellIndexPath()
 
         reconfigureDataSource(tableViewModel: tableViewModel, statuses: productImageActionHandler.productImageStatuses) { [weak self] in
-            guard let self = self else { return }
-
-            switch (indexPathBeforeReload, indexPathAfterReload) {
-            case (let indexPathBeforeReload?, nil):
-                self.tableView.deleteRows(at: [indexPathBeforeReload], with: .left)
-            case (nil, let indexPathAfterReload?):
-                self.tableView.insertRows(at: [indexPathAfterReload], with: .automatic)
-            default:
-                self.tableView.reloadData()
-            }
+            self?.tableView.reloadData()
```

Comment:
❓ If I understand the `reconfigureDataSource` code correctly, if we don't pass a `reloadClosure`, it will automatically reload the table view. And, we don't have to reload the tableView here. What do you think?

Reply:
Yup, I removed the trailing closure in ef512e6e45cf54f913e7028b38426580385785af.